platform/viewer/README 2.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
We will for sure have to eject. All or most of the tooling should still be what we're targeting, unless there's something in particular that jumps out at you? This may be as simple as keeping the `.webpack` folder, but simplifying the build commands